requirements.txt

```diff
@@ -17,7 +17,7 @@ jsonschema<2.6,>=2.5.0
 kombu==3.0.37
 lockfile<0.11,>=0.10.2
 mock
-mongoengine==0.11.0
+mongoengine==0.12.0
```

Comment:
Is this intentional?

Would need to check and verify that there are no backward incompatible changes, etc.